app/src/main/java/ly/loud/loudly/ui/feed/FeedPresenter.java

```diff
@@ -86,17 +94,22 @@ public void deletePost(@NonNull LoudlyPost post) {
         deleterModel.deletePostFromAllNetworks(post)
                 .subscribeOn(io())
                 .observeOn(mainThread())
-                .doOnCompleted(this::updatePosts)
+                .doOnCompleted(this::startUpdatePosts)
                 .subscribe();
     }
 
-    @CheckResult
-    @NonNull
-    private Single<SolidList<PlainPost>> loadPosts() {
-        return postLoadModel
-                .getPostsByIterval(loadMoreStrategyModel.getCurrentTimeInterval())
+    private boolean isAnyNetworkConnected() {
+        return coreModel.getConnectedNetworksModels().size() > 0;
+    }
+
+    private void loadPosts(@NonNull Action1<SolidList<PlainPost>> resultAction) {
+        postLoadModel.getPostsByInterval(loadMoreStrategyModel.getCurrentTimeInterval())
                 .subscribeOn(io())
-                .observeOn(mainThread());
+                .observeOn(mainThread())
+                .subscribe(
+                        resultAction,
+                        error -> executeIfViewBound(FeedView::onNetworkProblems)
```

Comment:
I think it's better to pass error to handler function, so it will show different messages